### PR TITLE
Incomplete descriptions

### PR DESCRIPTION
**Pull request by polinavino**
Some verbal descriptions of delegation rules

### CODE REVIEW

File: latex/ledger-spec.tex [hunk 1]
```diff
@@ -95,7 +95,6 @@
 \newcommand{\txbody}[1]{\fun{txbody}~ \var{#1}}
 \newcommand{\txfee}[1]{\fun{txfee}~ \var{#1}}
 \newcommand{\minfee}[2]{\fun{minfee}~ \var{#1}~ \var{#2}}
-\newcommand{\slotminus}[2]{\var{#1}~-_{s}~\var{#2}}
```
Comment (JaredCorduan) [inline latex/ledger-spec.tex]: did you mean to remove this?

File: latex/ledger-spec.tex [hunk 2]
```diff
@@ -154,14 +153,28 @@ \section{Cryptographic primitives}
...
+That is, their exact
+implementation remains open to interpretation, and we do not rely on
+any additional properties derived from the study or implementation of public key
+cryptography outside this work. The types and rules we give here are needed in
+order to guarantee certain security properties of the delegation process, which
+we discuss later.
+
+The cryptographic concepts required for the formal definition
+of delegation with witnesses include public-private key pairs, hash
+maps and signatures. The constraint we introduce states that a signature of
```
Comment (JaredCorduan) [inline latex/ledger-spec.tex]: did we want to say "one-way functions" to remove ambiguity about hashes?

File: latex/ledger-spec.tex [hunk 3]
```diff
@@ -296,19 +392,17 @@ \section{UTxO}
...
               \left(\fun{d}~pc~c\right) \cdot
             \left(d_{\min}+(1-d_{\min})\cdot e^{-\lambda\cdot\delta}\right)}
             & \text{otherwise}
         \end{cases}\\
-      &
-      \begin{array}{lr@{~=~}l}
-        \where &\fun{releasing}~\var{c} & \DeregKey{c} \lor \RetirePool{c}\\
-        & d_{\min},~\lambda & \fun{decay}~pc\\
-        &\delta & \slotminus{slot}{(allocs~(\cauthor c))}\\
-      \end{array}\\
```
Comment (JaredCorduan) [inline latex/ledger-spec.tex]: should this figure be changing?

File: latex/ledger-spec.tex [hunk 4]
```diff
@@ -320,6 +414,50 @@ \section{UTxO}
...
+For this reason, we make use here
+of the $\fun{txbody}$ map. This map returns only the data in the given
+transaction pertinent to the transfer of coins (i.e. the lists of inputs and outputs)
+
+The transaction fee $\fun{txfee}$ is the
+fee a transaction contributes to the system (this value depends only on
+the transaction itself).
+
+\begin{note}
+  Fees have no explicit output?
```
Comment (JaredCorduan) [inline latex/ledger-spec.tex]: correct. if the fee is explicitly specified in the transaction, then as long at the preservation of value condition is met (by the equality), then we do not need to specify an an explicit output for the fee

File: latex/ledger-spec.tex [hunk 5]
```diff
@@ -606,7 +860,8 @@ \subsection{Witnesses}
     \left(
       \begin{array}{r@{~\in~}lr}
         \var{utxo} & \UTxO & \text{UTxO}\\
-        \var{dwstate} & \DWState & \text{delegation witnesess state}\\
+        \var{dstate} & \DState & \text{delegation state}\\
+        \var{pstate} & \PState & \text{pool state}\\
```
Comment (JaredCorduan) [inline latex/ledger-spec.tex]: was this intentional?

File: latex/ledger-spec.tex (hunk 1 above)
Comment (polinavino) [reply latex/ledger-spec.tex]: No!

File: latex/ledger-spec.tex (hunk 2 above)
Comment (polinavino) [reply latex/ledger-spec.tex]: Yup!

File: latex/ledger-spec.tex (hunk 5 above)
Comment (polinavino) [reply latex/ledger-spec.tex]: I am not sure what happened! Not intentional

**Pull request closed without merging**